src/App.tsx

```diff
@@ -27,6 +30,7 @@ function App() {
   const [showCellId, setShowCellId] = React.useState<boolean>(false);
   const [showGoals, setShowGoals] = React.useState<boolean>(true);
   const [showGoalVectors, setShowGoalVectors] = React.useState<boolean>(false);
+  const [drawerOpen, setDrawerOpen] = React.useState<boolean>(false);
```

Comment:
```suggestion
  const [drawerOpen, setDrawerOpen] = React.useState<boolean>(true);
```
I think the user experience would be better if the drawer is initially open. Otherwise, it will not be clear what the user should do when they first open the visualizer.